finetuner/tuner/callback/early_stopping.py

```diff
@@ -47,7 +48,7 @@ def __init__(
             baseline.
         :param verbose: Wheter to log score improvement events.
         """
-        self._logger = logging.getLogger('finetuner.' + self.__class__.__name__)
+        self._logger = logging.getLogger("finetuner." + self.__class__.__name__)
```

Comment:
use single quotes to align with other jina projects

Reply:
oops, modified automatically by black

Reply:
when using local black do: `black -S filename.py`